### ISSUE TITLE
[Request] Being able to move the Daemon to a new IP without doing a fresh install.

### ISSUE THREAD

**Issue by alexbegt** [SpaceCP/Bugs#219]
I had to buy new IP's because I needed more so It ended up changing the current IP i was using for my Daemon, Is there anyway to change the Daemon IP to the new one without deleting the old Daemon + servers? As of posting this, I see no way to do this.

**Comment by ryan77627** [comment]
I do not believe there is a way to do this currently, but I thumbed this up because it is a good feature request.
